portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationEditSegment.testcase

```diff
@@ -36,6 +36,55 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-94874."
```

Comment:
You could add the bug description here, like this:
`@description = "LPS-151808 Bug: The button Metrics is shown for web content in Pending status"`

Reply:
ok!!